Applications/Spire/Source/Ui/IntegerSpinBox.cpp

```diff
@@ -32,6 +32,14 @@ void IntegerSpinBox::set_maximum(std::int64_t maximum) {
   m_spin_box->set_maximum(maximum);
 }
 
+std::int64_t IntegerSpinBox::get_step() const {
+  return m_spin_box->get_step().extract_signed_long_long();
```

Comment:
Check if there is a way to use a ```static_cast``` or some other exact conversion. It's not true that a ```signed long long``` always represents an ```std::int64_t```.